configs/m2cellCfg.yaml

```diff
@@ -0,0 +1,6 @@
+# m2cell configuration file.
+
+ControlServer:
+  host: "127.0.0.1"
+  port: 12678
+  threads: 1
```

Comment:
Please help to add the comment to explain the fields. Right now, the M2 cell code (LabVIEW) listens to the port 55555. In a latter time (around June or July), I will update the M2 cell code (LabVIEW) to listen to the command port 50000 and telemetry port 50001.